collectors/python.d.plugin/openldap/openldap.chart.py

```diff
@@ -139,8 +141,8 @@ def __init__(self, configuration=None, name=None):
 
         self.server = configuration.get('server', DEFAULT_SERVER)
         self.port = configuration.get('port', DEFAULT_PORT)
-        self.username = configuration.get('username')
-        self.password = configuration.get('password')
+        self.username = configuration.get('username', DEFAULT_USERNAME)
+        self.password = configuration.get('password', DEFAULT_PASSWORD)
```

Comment:
To be honest, I tried with `None` in many variations but I was too tired  to troubleshoot it when I tested. I can work more on it this afternoon:
```
2018-11-14 08:33:09: python.d DEBUG: plugin: main: module status: 'openldap' => [OK] (jobs: 1)
2018-11-14 08:33:09: python.d ERROR: openldap: openldap: check() unhandled exception: must be string, not None
2018-11-14 08:33:09: python.d ERROR: openldap: openldap: check() => [FAILED]
2018-11-14 08:33:09: python.d INFO: plugin: main: FINISHED
bash-4.2$
```

Reply:
hm, please add trace flag and run plugin in debug mode again

Reply:
Ok after some test, seems that simple bind method doesn't like 'None' values. But since we allow anonymous binds, simple bind os not necessary. So check out my last commit, and let me know if I need to change something further. It is tested and working.